src/components/settings/SettingsSSOTab/NewExistingSSOConfigs.jsx

```diff
@@ -0,0 +1,196 @@
+import {
+  CardGrid,
+  Skeleton,
+  useToggle,
+} from '@edx/paragon';
+import { useQuery, useQueryClient } from '@tanstack/react-query';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import LmsApiService from '../../../data/services/LmsApiService';
+import NewSSOConfigAlerts from './NewSSOConfigAlerts';
+import NewSSOConfigCard from './NewSSOConfigCard';
+
+const FRESH_CONFIG_POLLING_INTERVAL = 30000;
+const UPDATED_CONFIG_POLLING_INTERVAL = 2000;
+
+const NewExistingSSOConfigs = ({
+  configs, refreshBool, setRefreshBool, enterpriseId,
+}) => {
+  const [inactiveConfigs, setInactiveConfigs] = useState([]);
+  const [activeConfigs, setActiveConfigs] = useState([]);
+  const [inProgressConfigs, setInProgressConfigs] = useState([]);
+  const [untestedConfigs, setUntestedConfigs] = useState([]);
+  const [liveConfigs, setLiveConfigs] = useState([]);
+  const [notConfiguredConfigs, setNotConfiguredConfigs] = useState([]);
+  const [queryForTestedConfigs, setQueryForTestedConfigs] = useState(false);
+  const [queryForConfiguredConfigs, setQueryForConfiguredConfigs] = useState(false);
+  const [intervalMs, setIntervalMs] = React.useState(FRESH_CONFIG_POLLING_INTERVAL);
+  const [loading, setLoading] = useState(false);
+  const [showAlerts, openAlerts, closeAlerts] = useToggle(false);
+
+  const queryClient = useQueryClient();
+
+  useQuery({
+    queryKey: ['todos'],
```

Comment:
I see the queryKey `todos` in [tanstack sample code that uses Todo objects](https://tanstack.com/query/v4/docs/react/guides/query-keys), but should we be using something different here?

Reply:
yeah happy to make it something more related to our project, figured it was a key thing per project so I wasn't that worried but happy to add our own flair 👍